girder_tech_journal/models/journal.py

```diff
@@ -0,0 +1,81 @@
+from girder.models.model_base import AccessControlledModel
+
+from girder_tech_journal.constants import TechJournalSettingsDefault
```

Comment:
This line is complaining that TechJournalSettingsDefault doesn't exist in constants anymore.  Looks like this line should be removed.